x-pack/plugins/spaces/server/constants.ts

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export const SPACES_TELEMETRY_TYPE = 'spaces-telemetry';
```

Comment:
```suggestion
export const SPACES_USAGE_DATA_TYPE = 'spaces-usage-data';
```

Reply:
Renamed to `SPACES_USAGE_STATS_TYPE = 'spaces-usage-stats'`, see https://github.com/elastic/kibana/pull/83575#discussion_r529141568.